contrib/docker/src/DockerModule.scala

```diff
@@ -136,11 +130,11 @@ trait DockerModule { outer: JavaModule =>
       val pullLatestBase = IterableShellable(if (pull) Some("--pull") else None)
 
       val result = os
-        .proc(executable(), "build", tagArgs, pullLatestBase, dest)
+        .proc(executable(), "build", tagArgs, pullLatestBase, "-f", dockerfilePath, os.pwd)
```

Comment:
The "context" path is now `os.pwd` and we specify the Dockerfile path with `-f`

Reply:
Same here.

```suggestion
        .proc(executable(), "build", tagArgs, pullLatestBase, "-f", dockerfilePath, T.workspace)
```